docs/COMPILATION.MD

```diff
@@ -189,7 +189,34 @@ Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+**Using Visual Studio**
+
+#### Here you can see two files .vcxproj(marked with blue arrows) and one .sln project(marked with a red arrow)
+
+![alt text](img/projectFiles.png)
+
+You have to open .sln if you want to build both non GUI and GUI ccextractor or you can build them seperately opening .vcxproj files one by one.
+
+
+#### Further, if you receive a confirmation request about downloading from unsafe resource just click OK and go forward</h4>
+![alt text](img/Warning.png)
+
+#### Then if you opened .sln file you will have 2 projects on the right pannel GUI and non GUI and you can also open them separetely by opening .vcxproj files.</h4>
```

Comment:
This is a redundant line, better to remove it and add tree structure in next line.